include/tdx.h

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+
+#ifndef SHIM_TDX_H
+#define SHIM_TDX_H
+
+typedef struct tdEFI_TD_VERSION {
+	uint8_t Major;
+	uint8_t Minor;
+} __attribute__((packed)) EFI_TD_VERSION;
+
+typedef uint32_t EFI_TD_EVENT_LOG_BITMAP;
+typedef uint32_t EFI_TD_EVENT_LOG_FORMAT;
+typedef uint32_t EFI_TD_EVENT_ALGORITHM_BITMAP;
+typedef uint32_t EFI_TD_MR_INDEX;
+
+typedef struct tdEFI_TD_EVENT_HEADER {
+	//
+	// Size of the event header itself (sizeof(EFI_TD_EVENT_HEADER)).
+	//
+	uint32_t HeaderSize;
+	//
+	// Header version. For this version of this specification, the value shall be 1.
+	//
+	uint16_t HeaderVersion;
+	//
+	// Index of the MR that shall be extended.
+	//
+	EFI_TD_MR_INDEX MrIndex;
+	//
+	// Type of the event that shall be extended (and optionally logged).
+	//
+	uint32_t EventType;
+} __attribute__((packed)) EFI_TD_EVENT_HEADER;
+
+typedef struct tdEFI_TD_EVENT {
+	//
+	// Total size of the event including the Size component, the header and the Event data.
```

Comment:
Please check your line lengths.

Reply:
Thanks comment @frozencemetery, it was fixed at new commit https://github.com/rhboot/shim/commit/5f2abe426bb04fef2d2104f809260327af874c67, please help to review!